src/modules/presets.js

```diff
@@ -0,0 +1,33 @@
+function applyPreset(preset) {
+  const result = {};
+  switch (preset) {
+    case 'dial-once': {
+      if (['staging', 'production'].includes(process.env.NODE_ENV)) {
+        process.env.CONSOLE_LOGGING = 'false';
```

Comment:
can a preset be done without altering the env vars, I guess as a user I would really not like to be surprised that I set some env var to one value and code makes it to something else and I don't know about it. (because really I wanna use a simple version to start logging, not spend too much time analysing the internals of this lib)